flexfactor_tests.py

```
@@ -7577,9 +7802,23 @@ def _make_target(self, tmp):
                        capture_output=True)
         with open(os.path.join(tmp, "src.js"), "w", encoding="utf-8") as fh:
             fh.write("console.log('app');\n")
+        with open(os.path.join(tmp, "package.json"), "w", encoding="utf-8") as fh:
+            fh.write('{"name":"fixture","version":"1.0.0",'
+                     '"scripts":{"build":"node -e \\"process.exit(0)\\""}}')
         subprocess.run(["git", "-C", tmp, "add", "-A"], capture_output=True)
         subprocess.run(["git", "-C", tmp, "commit", "-q", "-m", "init"],
                        capture_output=True)
+        branch = subprocess.run(
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)